### ISSUE TITLE
Meaningful sequence (WCAG 1.3.2a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1225]
Images and captions are not consecutive in HTML. If the image is not marked as a figure with figcaption, the connection may be not clear to the screen reader user. Image and caption should be linked in HTML by using the mentioned elements.

https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437649-3/?language=en-/slide-719796-3#/slide-437649-3

**Comment by aoelen** [comment]
Because of the many changes in forms, I want to ask you to test 3, 4 and 6 again when the new version is released (this will be soon). 

For 5, I am not sure which link you mean. Could you explain this?

**Comment by VivienDZB** [comment]
I tried to recreate issue 5 with NVDA but I honestly don't know what I meant there. 😅

**Comment by VivienDZB** [comment]
And of course I'll test that again. Do you already have a specific release date?

**Comment by aoelen** [comment]
This shouldn't take too long. Currently, the code is ready to be released, I will let you know when it is released.
